lib/src/chip/chip.component.html

```diff
@@ -0,0 +1,6 @@
+<div class="custom-chip" [ngClass]="className" [attr.id]="id">
+    <div class="content"><ng-content></ng-content></div>
```

Comment:
Please use `select` to project the content. It should look like this. 
`<ng-content select="[chipContent]"></ng-content>`

https://stackoverflow.com/questions/52638718/multiple-ng-content
![image](https://user-images.githubusercontent.com/49055952/81644799-01b81f00-945b-11ea-8ae5-5910d3068c43.png)

Reply:
Don't see the need for this because we are rendering just one content.

Reply:
Exactly so you don't need an unecessary wrapper with class content which does nothing. Please use the example I gave you to refactor as this is the angular way and other components are following this logic as well.

Reply:
so, you mean something like this:
```suggestion
    <ng-content></ng-content>
```

Reply:
I am aware of the ng-content select Mario and i don't see the need for it here. You use select when you have multiple ng-content, which we don't in this component. It's just a simple chip component and regardless the user can have just one-content. So why the hassle of adding a select/target.

Reply:
Then you can at least omit the unnecessary wrapper